nwb_conversion_tools/json_schema_utils.py

```diff
@@ -57,11 +55,25 @@ def get_schema_from_method_signature(class_method, exclude=None):
     if exclude is None:
         exclude = []
     input_schema = get_base_schema()
+    annotation_json_type_map = dict(
+        bool="boolean",
+        str="string",
+        int="number",
+        float="number",
+        dict="object",
+        list="array"
+    )
+
     for param in inspect.signature(class_method).parameters.values():
         if param.name not in exclude + ['self']:
+            if param.name != 'self' and param.annotation:
```

Comment:
```suggestion
            if param.annotation:
```
This is handled by the line above

Reply:
Ah, yes. I had it outside that statement then copied it in but forgot to remove it after. Thanks!